### DISCUSSION TITLE
Help Starting the dev server

### DISCUSSION THREAD

**Discussion by gordonscottkolb** [TEAMMATES/teammates#12297] [Help requests] [answered]
Hello! I am new to open source development! I am following the docs to set up the development environment and to manage the dev server (both front end and back end). I was able to start the dev server for the front end successfully, however, I am having some trouble with the back end. When I run "./gradlew serverRun", it seems to hang on ":serverRun" at "85% executing". 

I reran with --debug to view some more output. The process seems to be locking and unlocking daemon addresses but making no progress. Does anyone have an idea about what may be wrong?  I have attached a screenshot of the last few lines of output in my terminal. I have followed all of the steps in the "setting up a development environment" and I have docker running. 

Thank you for the help! 

<img width="1071" alt="Screen Shot 2023-03-28 at 4 47 53 PM" src="https://user-images.githubusercontent.com/56280856/228363165-0d991e17-237b-4a02-b5d0-a2341ea867e2.png">

**Comment by domlimm** [comment, marked as answer]
@gordonscottkolb Hello! Thanks for reaching out to us and your interest! This is the expected outcome on running the backend. You could search under the Issues tab for `85%` and you should get some results on them that others have encountered before and we've clarified on it. Let us know if you have any other questions 👍🏻

**Reply by gordonscottkolb** [reply]
Thank you for letting me know, I got it to work now!

**Reply by domlimm** [reply]
Awesome!
